vlasovsolver/cpu_acc_load_blocks.cpp

```diff
@@ -44,252 +44,505 @@ void loadColumnBlockData(
+         values[i_pcolumnv_b(0, 5, block_k, n_blocks)] = Vec({data[5], data[69], data[133], data[197], data[261], data[325], data[389], data[453], data[13], data[77], data[141], data[205], data[269], data[333], data[397], data[461], data[21], data[85], data[149], data[213], data[277], data[341], data[405], data[469], data[29], data[93], data[157], data[221], data[285], data[349], data[413], data[477], data[37], data[101], data[165], data[229], data[293], data[357], data[421], data[485], data[45], data[109], data[173], data[237], data[301], data[365], data[429], data[493], data[53], data[117], data[181], data[245], data[309], data[373], data[437], data[501], data[61], data[125], data[189], data[253], data[317], data[381], data[445], data[509]});
+         values[i_pcolumnv_b(0, 6, block_k, n_blocks)] = Vec({data[6], data[70], data[134], data[198], data[262], data[326], data[390], data[454], data[14], data[78], data[142], data[206], data[270], data[334], data[398], data[462], data[22], data[86], data[150], data[214], data[278], data[342], data[406], data[470], data[30], data[94], data[158], data[222], data[286], data[350], data[414], data[478], data[38], data[102], data[166], data[230], data[294], data[358], data[422], data[486], data[46], data[110], data[174], data[238], data[302], data[366], data[430], data[494], data[54], data[118], data[182], data[246], data[310], data[374], data[438], data[502], data[62], data[126], data[190], data[254], data[318], data[382], data[446], data[510]});
+         values[i_pcolumnv_b(0, 7, block_k, n_blocks)] = Vec({data[7], data[71], data[135], data[199], data[263], data[327], data[391], data[455], data[15], data[79], data[143], data[207], data[271], data[335], data[399], data[463], data[23], data[87], data[151], data[215], data[279], data[343], data[407], data[471], data[31], data[95], data[159], data[223], data[287], data[351], data[415], data[479], data[39], data[103], data[167], data[231], data[295], data[359], data[423], data[487], data[47], data[111], data[175], data[239], data[303], data[367], data[431], data[495], data[55], data[119], data[183], data[247], data[311], data[375], data[439], data[503], data[63], data[127], data[191], data[255], data[319], data[383], data[447], data[511]});
+   #else // Fell through, never fall into this particular pit again
+   	std::cerr << "Undefined VECTORCLASS flag or implementation missing in loadColumnBlockData() before " << __FILE__ << ":" << __LINE__ << std::endl;
```

Comment:
This line as an example: I don't like indentation that's not a multiple of 3 spaces...